src/redisearch.h

```diff
@@ -293,6 +293,9 @@ typedef struct RSIndexResult {
 
   /* Relative weight for scoring calculations. This is derived from the result's iterator weight */
   double weight;
+
+  /* Option: maintains explanation for how was the score calculated */
+  char *scoreExplainStr;
```

Comment:
This belongs more properly in `SearchResult`

Reply:
I'm using RSIndexResult as a vehicle into the scorers and recursive function. That is impossible with SearchResult.
The alternative is to have a second struct that will be passed between the functions. This seem less readable and less convenient.